plugin/src/main/java/net/aufdemrand/denizen/objects/properties/item/ItemPotion.java

```diff
@@ -311,7 +311,7 @@ public void adjust(Mechanism mechanism) {
         // @input dList
         // @description
         // Sets the potion's potion effect(s).
-        // Input is a formed like: Effect,Upgraded,Extended(,Color)|Type,Amplifier,Duration,Ambient,Particles(,Icon)|...
+        // Input is a formed like: Type,Upgraded,Extended(,Color)|Effect,Amplifier,Duration,Ambient,Particles(,Icon)|...

```

Comment:
Why was this change made?

Reply:
I felt like it better emphasized the difference between potion *types* (base potion) and potion *effects* (has been historically synonymous with status effects). Plus, it better fits with the two server list tags that make that distinction clear (`<server.list_potion_types>` returns a list of potion types, and `<server.list_potion_effects>` returns a list of potion effects).

Reply:
You would have to make significant changes to the rest of the `ItemPotion` file to try to make an idea like that work. Throughout the file, `type` and `effect` are used to mean the same concept, and `Effect` is used at various points for the base, and `Type` used for the list of effects. See for ref `
                // @attribute <i@item.potion_effect[<#>].type>`
And as well
```
        // @attribute <i@item.potion_base>
        // Returns the potion effect on this item.
        // In the format Effect,Level,Extended,Splash,Color
```

Reply:
Oh, I didn't see that. Can probably open a separate PR for that when I finish up the rest of my potion changes (ItemPotion and EntityPotion), since I believe I fixed that as part of that set of changes.